test/server.jl

```diff
@@ -36,24 +36,24 @@ end
     socket = connect(port)
     lintbyserver(socket)
     response = ""
-    line = ""
-    while line != "\n"
+    line = "."
+    while !isempty(line)
```

Comment:
This change was needed to prevent a hang. On v0.6, `readline` changed to remove trailing newlines.